ddl-scripts/db2/db2-pim-schema.sql

```diff
@@ -0,0 +1,80 @@
+create sequence MIG_REP_ID_SEQ start with 1 increment by 50;
+create sequence MIGRATION_ID_SEQ start with 1 increment by 50;
+create sequence PLAN_ID_SEQ start with 1 increment by 50;
+
+create table migration_report_logs (
+   report_id bigint not null,
```

Comment:
the first line of each `create` seems to be one space short

Reply:
done